tools/bpf2c/bpf2c.cpp

```diff
@@ -51,6 +72,9 @@ main(int argc, char** argv)
                 {"--dll",
                  {"Generate code for a Windows DLL",
                   [&](std::vector<std::string>::iterator&) { type = output_type::UserPE; }}},
+                {"--verify",
```

Comment:
I think --verify should be the default option, and we should instead have a --no-verify option. Also, I think --no-verify option should be available only for debug builds.